src/main/java/com/adobe/aem/compgenerator/utils/HTMLUtils.java

```diff
@@ -0,0 +1,115 @@
+package com.adobe.aem.compgenerator.utils;
+
+import com.adobe.aem.compgenerator.Constants;
+import com.adobe.aem.compgenerator.models.GenerationConfig;
+import com.adobe.aem.compgenerator.models.Property;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class HTMLUtils {
+
+    public static String renderHtml(GenerationConfig generationConfig) {
+        List<Property> props = getAllProperties(generationConfig);
+        String slingModel = StringUtils.uncapitalize(generationConfig.getJavaFormatedName()) + "Model";
+        StringBuilder renderedHtml = new StringBuilder();
+
+        for (Property prop : props) {
+            String label = getLabel(prop);
+
+            if (Constants.TYPE_CHECKBOX.equals(prop.getType())) {
+                label = prop.getAttributes().get("text");
```

Comment:
Should this logic be in `getLabel()`?  Seems we consolidated most logic there, but then kept this out.

Reply:
Changed.